### PR TITLE
LRQA-78596 | Test Fix | Master 

### PR DESCRIPTION
**Pull request by sirfabios**
https://issues.liferay.com/browse/LRQA-78596

### CODE REVIEW

**Comment by sirfabios** [comment]
ci:test:sf

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/macros/ProcessBuilderKaleoDesignerReact.macro [hunk 1]
```diff
@@ -614,11 +614,18 @@ definition {
...
+				key_inputOption = "Type",
+				locator1 = "ProcessBuilderKaleoDesignerReact#PROPERTIES_TYPE_TEXTAREA",
+				value1 = "${type}");
+		}
+
+		if (isSet(script)) {
+			AssertElementPresent(
+				key_inputOption = "Script",
+				locator1 = "ProcessBuilderKaleoDesignerReact#PROPERTIES_SCRIPT_TEXTAREA",
+				value1 = "${script}");
```
Comment (evanilson-santana) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/macros/ProcessBuilderKaleoDesignerReact.macro:628]: These changes are the same as this: [PR](https://github.com/liferay-bpm-qa/liferay-portal/pull/163/files)

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/macros/ProcessBuilderKaleoDesignerReact.macro [hunk 2]
```diff
@@ -683,7 +690,9 @@ definition {
 			key_inputOption = "Description",
 			locator1 = "ProcessBuilderKaleoDesignerReact#PROPERTIES_INPUT");
 
-		AssertElementPresent(locator1 = "ProcessBuilderKaleoDesignerReact#PROPERTIES_TEMPLATE_TEXTAREA");
+		AssertElementPresent(
+			key_inputOption = "Type",
+			locator1 = "ProcessBuilderKaleoDesignerReact#PROPERTIES_TYPE_TEXTAREA");
```
Comment (evanilson-santana) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/macros/ProcessBuilderKaleoDesignerReact.macro:695]: The same thing here

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/macros/ProcessBuilderKaleoDesignerReact.macro [hunk 3]
```diff
@@ -708,10 +717,18 @@ definition {
...
+					value1 = "${type}");
+			}
+
+			if (isSet(script)) {
 				Type(
-					locator1 = "ProcessBuilderKaleoDesignerReact#PROPERTIES_TEMPLATE_TEXTAREA",
-					value1 = "${actionsTemplateValue}");
+					key_inputOption = "Script",
+					locator1 = "ProcessBuilderKaleoDesignerReact#PROPERTIES_SCRIPT_TEXTAREA",
+					value1 = "${script}");
```
Comment (evanilson-santana) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/macros/ProcessBuilderKaleoDesignerReact.macro:731]: The same thing here

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/paths/ProcessBuilderKaleoDesignerReact.path [hunk 4]
```diff
@@ -556,6 +556,16 @@
...
+<tr>
+	<td>PROPERTIES_SCRIPT_TEXTAREA</td>
+	<td>//label[contains(., 'Script')]/..//textarea</td>
+	<td></td>
+</tr>
+<tr>
+	<td>PROPERTIES_TYPE_TEXTAREA</td>
+	<td>//label[@for='type']/..//select[@id='type']</td>
+	<td></td>
+</tr>
```
Comment (evanilson-santana) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/paths/ProcessBuilderKaleoDesignerReact.path:568]: And here

**Review by evanilson-santana — CHANGES_REQUESTED** [review]
Hey @sirfabios can you combine the changes of this [PR](https://github.com/liferay-bpm-qa/liferay-portal/pull/163) here? You can create two commits for both test fixes.

Please see my comments and let me know if you have any questions.

**Comment by sirfabios** [comment]
ci:test:sf

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase [hunk 5]
```diff
@@ -1005,7 +1005,6 @@ definition {
 	@description = "Verify that the user can delete action with the trash icon at the bottom right corner"
 	@priority = "4"
 	test CanDeleteAction {
-		property portal.upstream = "quarantine";
```
Comment (andreefarias) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase:1004]: Hey @sirfabios,

Next time, you can use the property on a different commit.

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase [hunk 6]
```diff
@@ -346,7 +346,6 @@ definition {
 	@description = "Verify that the user is able to add a Script to define the Action"
 	@priority = "3"
 	test CanAddScriptToDefineAction {
-		property portal.upstream = "quarantine";
```
Comment (andreefarias) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase:349]: Hey @sirfabios,

Next time, you can use the property on a different commit.

**Comment by andreefarias** [comment]
ci:forward

**Pull request closed without merging**